SConstruct

```diff
@@ -393,16 +393,20 @@ def config_base(env):
         env.Append(CPPPATH=[os.path.join(profile_jemalloc, 'include')])
         env.Append(LINKFLAGS=['-Wl,-rpath,' + os.path.join(profile_jemalloc, 'lib')])
 
-def add_static_libs(env, libs):
+def add_static_libs(env, static_libs, dyn_libs=None):
     if not 'HASSTATICLIBS' in env:
         env['HASSTATICLIBS'] = True
         env['STATICLIBS'] = ''
-        env.Replace(LINKCOM=env['LINKCOM'] + " -Wl,-Bstatic $STATICLIBS")
-    c = env['STATICLIBS']
-    if c == None: c = ''
-    for f in libs:
-        c += ' -l' + f 
-    env['STATICLIBS'] = c
+        env['DYNAMICLIBS'] = ''
```

Comment:
Why bother initializing `env['STATICLIBS'] and env['DYNAMICLIBS']` here when you're about to initialize them on line 406 anyway? Or do you explicitly want to make sure they're _empty_ in this case?